src/argilla/client/sdk/v1/datasets/api.py

```diff
@@ -191,12 +193,15 @@ def get_records(
 
     params = {"include": ["responses", "suggestions"], "offset": offset, "limit": limit}
 
-    if response_status is not None:
+    if response_status:
```

Comment:
I think `is not None` is more consistent here, is there any reason for this change?

Reply:
just to avoid passing a []